__tests__/app.test.js

```diff
@@ -40,6 +40,48 @@ describe('GET /api', () => {
     });
 });
 
+describe('GET /api/articles/:article_id', () => {
+    it('200: responds with article object with correct properties', () => {
+        const testId = 1
+        return request(app)
+        .get(`/api/articles/${testId}`)
+        .expect(200)
+        .then(({body}) => {
+            expect(Object.keys(body.article)).toHaveLength(8)
```

Comment:
You are specifically asking for the article_id of 1 and all the corresponding data. I would expect to see that data. This test would pass if I asked for article 1 but was served article 12. Just because they have the same keys.